Travis is saying that tests in workers/interfaces.worker.html no longer exists and is causing tests to fail

```
--- FAIL: TestResultsTestPaths/Manifest_@_a12f37b/workers (0.00s)

            --- FAIL: TestResultsTestPaths/Manifest_@_a12f37b/workers/workers/interfaces.worker.html (0.00s)

                test_paths_test.go:102: 

                    	Error Trace:	test_paths_test.go:102

                    	Error:      	Should be true

                    	Test:       	TestResultsTestPaths/Manifest_@_a12f37b/workers/workers/interfaces.worker.html

                    	Messages:   	workers/interfaces.worker.html is not a test path found in the manifest @ a12f37b
```

--- a/css/css-flexbox/META.yml
+++ b/css/css-flexbox/META.yml
@@ -1,9 +1,6 @@
 links:
 - product: firefox
+  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1543267
   results:
   - test: position-absolute-001.html
-  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1543267
-- product: firefox
-  results:
   - test: position-absolute-002.html
-  url: https://bugzilla.mozilla.org/show_bug.cgi?id=1543267

--- a/dom/META.yml
+++ b/dom/META.yml
@@ -2,13 +2,13 @@ links:
 - product: firefox
   results:
   - subtest: 'Document interface: attribute origin'
-    test: interfaces.html?exclude=Node
+    test: idlharness.window.html?exclude=Node
   - subtest: 'Document interface: xmlDoc must inherit property "origin" with the proper
       type (3)'
-    test: interfaces.html?exclude=Node
+    test: idlharness.window.html?exclude=Node
   - subtest: 'Document interface: new Document() must inherit property "origin" with
       the proper type (3)'
-    test: interfaces.html?exclude=Node
+    test: idlharness.window.html?exclude=Node
   url: https://bugzilla.mozilla.org/show_bug.cgi?id=931884
 - product: firefox
   results:

--- a/html/dom/META.yml
+++ b/html/dom/META.yml
@@ -2,11 +2,11 @@ links:
 - product: firefox
   results:
   - subtest: 'Window interface: window must have own property "window"'
-    test: interfaces.https.html?include=(Document|Window)
+    test: idlharness.https.html?include=(Document|Window)
   - subtest: 'Window interface: window must have own property "document"'
-    test: interfaces.https.html?include=(Document|Window)
+    test: idlharness.https.html?include=(Document|Window)
   - subtest: 'Window interface: window must have own property "top"'
-    test: interfaces.https.html?include=(Document|Window)
+    test: idlharness.https.html?include=(Document|Window)
   - subtest: 'Window interface: window must have own property "location"'
-    test: interfaces.https.html?include=(Document|Window)
+    test: idlharness.https.html?include=(Document|Window)
   url: https://bugzilla.mozilla.org/show_bug.cgi?id=1510437

--- a/workers/META.yml
+++ b/workers/META.yml
@@ -1,6 +1,0 @@
-links:
-  - product: chrome
-    url: https://bugs.chromium.org/p/chromium/issues/detail?id=114475
-    results:
-    - status: FAIL
-      test: interfaces.worker.html

